module-system/module-implementations/sov-evm/src/evm/conversions.rs

```diff
@@ -25,3 +30,59 @@ impl From<ReVmAccountInfo> for AccountInfo {
         }
     }
 }
+
+impl From<BlockEnv> for ReVmBlockEnv {
+    fn from(block_env: BlockEnv) -> Self {
+        Self {
+            number: U256::from_le_bytes(block_env.number),
+            coinbase: B160::from_slice(&block_env.coinbase),
+            timestamp: U256::from_le_bytes(block_env.timestamp),
+            // TODO: handle difficulty
+            difficulty: U256::ZERO,
+            prevrandao: block_env.prevrandao.map(|r| B256::from_slice(&r)),
+            basefee: U256::from_le_bytes(block_env.basefee),
+            gas_limit: U256::from_le_bytes(block_env.gas_limit),
+        }
+    }
+}
+
+impl From<AccessListItem> for (B160, Vec<U256>) {
```

Comment:
What does this tuple mean? Maybe we should create a regular struct with field names?

Reply:
This is why we have `AccessListItem`, but `revm` expects the above tuple (it is part of the interface and we can't change it) this is why we provide this conversions.